### ISSUE TITLE
Simple package installation functionality, tests, docs

### ISSUE THREAD

**Issue by sushain97** [apertium/apertium-python#39]
something like `apertium.install('en-es')`.

**Comment by sushain97** [comment]
Still missing tests I think?

**Comment by Vaydheesh** [comment]
Unittests for wrapper?

**Comment by sushain97** [comment]
For installation.

**Comment by Vaydheesh** [comment]
Tests for installation of language packages?

**Comment by sushain97** [comment]
Yes.

**Comment by sushain97** [comment]
I think you added these, right? Hopefully there are installations of packages that use hfst stuff as well as lttoolbox?

**Comment by Vaydheesh** [comment]
> I think you added these, right?

yes. The installation-tests needs some improvement to ensure that packages are being installed properly and not the existing ones aren't used

> Hopefully there are installations of packages that use hfst stuff as well as lttoolbox?

IIRC `hfst-proc` isn't wrapped so hfst-python isn't installed.
